index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
```

Comment:
<!-- Add a suggestion to improve accessibility -->
<tr>
    <td><label for="game-over-message" hidden>Game over</label></td>
    <td><input type="text" id="game-over-message" value="Game over, refresh to play again 🧑‍💻 🤖!" aria-live="assertive" hidden></td>
</tr>